sdk/rust/oak_tests/certs/ca.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Could / should we use symlinks instead?

Reply:
I'd say no, because there isn't any requirement that the files here have to be the same as the ones in `/examples/certs/`.  

(At some point I'll replace the contents here, but the regeneration instructions aren't complete)